README.md

```diff
@@ -145,10 +141,11 @@ And example command to deploy to the `e79518-dev` namespace is:
 
 ```bash
 helm template bcwallet ./devops/charts/controller
---set-string tenant_id=123-456-789 \
---set-string tenant_api_key=abc-def-ghi \
---set-string redis_url=redis://redis:6379/0 \
```

Comment:
We don't need a redis URL env variable anymore?

Reply:
No, I get it from a secret the ops team have already created. Still needed for local dev though.